### PR TITLE
Add RISCV binary-to-string conversion

### PR DESCRIPTION
**Pull request by cayb0rg**
Adds the binary-encoded instruction to string converter for RISC-V instructions. Also changes `Instruction` structs to `MipsInstruction `and `RiscInstruction` to differentiate between the two. Adds missing commas in MIPS instruction disassembly.

### CODE REVIEW

File: src/bin/main.rs [hunk 1]
```diff
@@ -38,7 +41,7 @@ use yew_agent::Spawnable;
 // To load in the Fibonacci example, uncomment the CONTENT and fib_model lines
 // and comment the code, language, and text_model lines. IMPORTANT:
 // rename fib_model to text_model to have it work.
-const CONTENT: &str = include_str!("../../static/assembly_examples/fibonacci.asm");
+const CONTENT: &str = include_str!("../../static/assembly_examples/riscv_fib_recursive.asm");
```
Comment (brooksmckinley) [inline src/bin/main.rs:44]: Since MIPS is still the default, can we switch this back to the original?

File: static/assembly_examples/syscalls.asm
Comment (brooksmckinley) [inline static/assembly_examples/syscalls.asm:1]: What is this demonstrating? It seems like it's just halting.

**Review by Geetis — APPROVED** [review]
Other than Brooks' requested change, looks good

File: src/bin/main.rs (hunk 1 above)
Comment (b0x-Cub3d) [reply src/bin/main.rs:44]: May I also add, if it's possible to toggle between the two programs when the language is selected? And if you get that working, you guys might have an idea on creating drop-down menu to load in the examples programs. Just a suggestion.

File: static/assembly_examples/syscalls.asm
Comment (cayb0rg) [reply static/assembly_examples/syscalls.asm:1]: Oop. This was the equivalent of scrap paper, forget you ever saw this.

**Pull request merged**